src/plugins/index_pattern_management/public/components/edit_index_pattern/create_edit_field/create_edit_field.tsx

```diff
@@ -95,24 +99,33 @@ export const CreateEditField = withRouter(
 
     if (field) {
       return (
-        <EuiFlexGroup direction="column">
-          <EuiFlexItem>
-            <IndexHeader indexPattern={indexPattern} defaultIndex={getConfig('defaultIndex')} />
-          </EuiFlexItem>
-          <EuiFlexItem>
-            <FieldEditor
-              indexPattern={indexPattern}
-              field={field}
-              helpers={{
-                getConfig,
-                getHttpStart: services.getHttpStart,
-                fieldFormatEditors,
-                redirectAway,
-                docLinksScriptedFields: services.docLinksScriptedFields,
-              }}
-            />
-          </EuiFlexItem>
-        </EuiFlexGroup>
+        <EuiPanel paddingSize={'l'}>
+          <EuiFlexGroup direction="column">
+            <EuiFlexItem>
+              <IndexHeader
+                indexPattern={indexPattern}
+                defaultIndex={services.uiSettings.get('defaultIndex')}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <FieldEditor
+                indexPattern={indexPattern}
+                field={field}
+                helpers={{
```

Comment:
nit: I don't like how it looks now. Maybe we should rename it back to `services` / `dependencies`?